_includes/integrations/react-quick-start.md

```diff
@@ -18,69 +18,65 @@ This procedure requires:
 
 ### Procedure
 
-1. On a command line or command prompt, install the [Create React App](https://github.com/facebook/create-react-app) package.
+1. Use the the [Create React App](https://github.com/facebook/create-react-app) package to create a new React project named `tinymce-react-demo`.
 
     ```sh
-    $ npm install -g create-react-app
+    $ npx create-react-app tinymce-react-demo
     ```
 
-2. Create a new React project named `tinymce-react-demo`.
-
-    ```sh
-    $ create-react-app tinymce-react-demo
-    ```
-
-3. Change into the newly created directory.
+2. Change into the newly created directory.
 
     ```sh
     $ cd tinymce-react-demo
     ```
 
-4. Install the `tinymce-react` package and save it to your `package.json` with `--save`.
+3. Install the `tinymce-react` package and save it to your `package.json` with `--save`.
 
     ```sh
     $ npm install --save @tinymce/tinymce-react
     ```
 
-5. Using a text editor, open `/path/to/tinymce-react-demo/src/App.js` and replace the contents with:
+4. Using a text editor, open `./src/App.js` and replace the contents with:
 
     ```jsx
-    import React from 'react';
+    import React, { useRef } from 'react';
     import { Editor } from '@tinymce/tinymce-react';
 
-    class App extends React.Component {
-      handleEditorChange = (content, editor) => {
-        console.log('Content was updated:', content);
-      }
-
-      render() {
-        return (
-          <Editor
-            initialValue="<p>This is the initial content of the editor</p>"
-            init={% raw %}{{{% endraw %}
-              height: 500,
-              menubar: false,
-              plugins: [
-                'advlist autolink lists link image charmap print preview anchor',
-                'searchreplace visualblocks code fullscreen',
-                'insertdatetime media table paste code help wordcount'
-              ],
-              toolbar:
-                'undo redo | formatselect | bold italic backcolor | \
-                alignleft aligncenter alignright alignjustify | \
-                bullist numlist outdent indent | removeformat | help'
-            {% raw %}}}{% endraw %}
-            onEditorChange={this.handleEditorChange}
-          />
-        );
-      }
+    export default function App() {
+      const edRef = useRef(null);
+      const log = () => {
+        if (edRef.current) {
+          console.log(edRef.current.getContent());
+        }
+      };
+      return (
+        <>
```

Comment:
I haven't used the shorthand before but is this indentation level common?

Reply:
It's probably fine either way as long as I was consistent. I was not however so I have indented all of them now.